README.md

```diff
@@ -178,8 +177,8 @@ like the native ES5 `JSON` object.
 
 To use from Node:
 
-```
-npm install json5
+```sh
+$ npm install json5
```

Comment:
The lack of `$` in these bash snippets was actually intentional: it lets you copy-paste directly.

(There have been some fun blog posts written on this topic, and I just spent an unreasonable amount of time trying to find one, but couldn't. Hard to search for them...)

So would you mind reverting the additions of `$` in these snippets? Thanks!